### DISCUSSION TITLE
Add HTTP Toolkit to Network Tools

### DISCUSSION THREAD

**Discussion by saulpanders** [OWASP/mastg#2665] [Ideas]
[HTTP Toolkit](https://httptoolkit.com/android/)

Its another alternative to Burp proxy, it can intercept HTTP traffic for android devices. It also has some automated abilities to get around certain certificate pinning issues by automatically injecting a system CA on install.

I dont have any affiliation with the tool, but used it recently and found it interesting

**Comment by cpholguera** [comment]
Thanks @saulpanders, added it as an issue to be evaluated 

https://github.com/OWASP/owasp-mastg/issues/2793
